person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
+
+  @person_count ||= 0
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = @person_count += 1
```

Comment:
## OPTIONAL 
- [ ] Good job incrementing the id for each person 👏🏿 here is another way to do this, you can also make the  @id = Random.rand(1..1000)